docs/settings/triggers/snowboy.md

```diff
@@ -4,23 +4,44 @@ You can create your magic word by connecting to [Snowboy](https://snowboy.kitt.a
 
 Once downloaded, place the file in your personal config folder and configure snowboy in your [your settings](../settings.md) following the table bellow
 
-| parameter   | required | type   | default | choices         | comment                                                                                          |
-|-------------|----------|--------|---------|-----------------|--------------------------------------------------------------------------------------------------|
-| pmdl_file   | TRUE     | string |         |                 | Path to the snowboy model file. The path can be absolute or relative to the brain file           |
-| sensitivity | FALSE    | string | 0.5     | between 0 and 1 | Increasing the sensitivity value lead to better detection rate, but also higher false alarm rate |
+| parameter      | required | type   | default | choices         | comment                                                                                          |
+|----------------|----------|--------|---------|-----------------|--------------------------------------------------------------------------------------------------|
+| keyword_file   | TRUE     | string |         |                 | Path to the snowboy model file. The path can be absolute or relative to the brain file           |
+| keywords       | FALSE    | string |         |                 | If you want to use multiple keywords                                                             |
+| sensitivity    | FALSE    | string | 0.5     | between 0 and 1 | Increasing the sensitivity value lead to better detection rate, but also higher false alarm rate |
+| apply_frontend | FALSE    | string | False   | True/False      | Some universal models (umdl files) need apply_frontend to work properly                          |        
```

Comment:
This doc should appears in our doc: https://github.com/Kitt-AI/snowboy/pull/419/files

Reply:
done